src/main/native/smithwaterman/avx512-smithwaterman.h

```diff
@@ -0,0 +1,7 @@
+#include <stdint.h>
+
+#undef SIMD_ENGINE
+#define SIMD_ENGINE avx512
+
+#include "avx512-functions.h"
+#include "PairWiseSW.h"
```

Comment:
Add newline at end of file.

Reply:
Thanks. I have incorporated the changes you requested and updated the pull request below.